src/resolvers/index.ts

```diff
@@ -22,10 +23,27 @@ const noPerms = {
 
 export const resolvers: Resolvers<Context> = {
   Query: {
-    me: (_, _params, ctx) => {
+    me: async (_, _params, ctx) => {
+      let username;
       if (!ctx.session.user) return null;
-      console.log(ctx.session.user.username);
-      return null;
+      else if (!ctx.session.user.username) return null;
+      else username = ctx.session.user.username;
+
+      return await getMe(username);
```

Comment:
You can remove the `let username` and return directly by removing the `else` here and just passing in the `getMe(ctx.session.user.username)`.